src/solvers/primal_dual_semismooth_Newton.jl

```diff
@@ -0,0 +1,378 @@
+using SparseArrays
```

Comment:
Then we also have to adapt the `Project.toml` and add it as a dependency including a compat entry. Otherwise the JuliaRegistry will deny the update. The easiest to add it to the Project.toml is by activating the environment (in the main folder this time) and add it there. But the compat-entry (line in the `[compat]` section of the file has to be done by hand, stating which version is required (of SparseArrays) to work here.

Reply:
I've added sparseArrays locally now and the project.toml has changed.

For compute, if I ask for ] status of Sparsearrays I don't get a version... Is this because it is a standard library? (https://docs.julialang.org/en/v1/stdlib/SparseArrays/) Don't we need a version then (like with Dates)?

Please share thoughts on how to proceed :)

Reply:
Ah, I did not remember it was a standard Library. If the tests with Julia 1.6 run fine (we currently require Julia 1.6 at least), then the add should be all thats necessary (that is committing the new Project.toml as well, for sure).

Reply:
I changed the toml, but GitHub might have missed it. Could you check if you see any changes

Reply:
It was already in the project.toml on master, so nothing changed

Reply:
Oh, I overlooked that, sorry.